src/main/java/org/jenkinsci/test/acceptance/junit/WithPlugins.java

```diff
@@ -98,9 +98,6 @@
         @Inject @Named("ExercisedPluginReporter")
         ExercisedPluginsReporter pluginReporter;
 
-        @Inject(optional=true) @Named("neverReplaceExistingPlugins")
-        boolean neverReplaceExistingPlugins;
```

Comment:
Note this is used in `FallbackConfig`'s provider

Reply:
Shame on me, my search keyword was wrong, fixed